### PR TITLE
sway: 0.14.0 -> 0.15.0

### PR DESCRIPTION
**Pull request by dywedir**
###### Motivation for this change

https://github.com/swaywm/sway/releases/tag/0.15.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by c0bw3b** [comment]
Related: #31371

File: pkgs/applications/window-managers/sway/default.nix [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   name = "sway-${version}";
-  version = "0.14.0";
+  version = "0.15.0";
```
Comment (gnidorah) [inline pkgs/applications/window-managers/sway/default.nix]: @dywedir You should also update wlc to 0.0.11 in this PR.
> We always depend on the latest version of wlc (master). We ship sway and wlc releases together.
via https://github.com/swaywm/sway/pull/1456#issuecomment-342142666

Comment (dywedir) [reply pkgs/applications/window-managers/sway/default.nix]: @gnidorah thanks!

**Comment by primeos** [comment]
Sway 0.15 works fine and the `wlc` upgrade doesn't break anything, thanks :smile: .

**Comment by concatime** [comment]
Using the channel `nixos-unstable-small`, I get this warnings when running sway 0.15.0:

1. `Sway does not have CAP_SYS_PTRACE`, which is related to this solved [issue](//github.com/swaywm/sway/issues/1346).

2. `Connection is null, cannot init SNI host`, which dispears when I use [`dbus-launch sway`](//github.com/NixOS/nixpkgs/pull/30493#issuecomment-338224367)

**Comment by gnidorah** [comment]
@concatime Here's the current way https://github.com/concatime/dotfiles/pull/1

**Comment by concatime** [comment]
Thank you bro!

**Pull request merged**